<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>681</width>
      <height>421</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_usb">
     <attribute name="title">
      <string>Start</string>
     </attribute>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>1</y>
        <width>421</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_connect">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>98</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_link">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>98</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Open dev</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_send">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>98</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_DevNum">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>60</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_usbMessage">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>30</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_sendMessage">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>98</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_close1">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>340</y>
        <width>98</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>230</y>
        <width>411</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>120</y>
        <width>401</width>
        <height>251</height>
       </rect>
      </property>
      <property name="title">
       <string>Pid Controll</string>
      </property>
      <widget class="QDoubleSpinBox" name="dSpinBox_D">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>210</y>
         <width>62</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QSlider" name="verticalSlider_D">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>40</y>
         <width>29</width>
         <height>160</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="dSpinBox_P">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>62</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QSlider" name="verticalSlider_I">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>40</y>
         <width>29</width>
         <height>160</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="dSpinBox_I">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>210</y>
         <width>62</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QSlider" name="verticalSlider_P">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>29</width>
         <height>160</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="dSpinBox_PWM">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>210</y>
         <width>71</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QDial" name="dial_Temp">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>140</y>
         <width>50</width>
         <height>64</height>
        </rect>
       </property>
      </widget>
      <widget class="QDial" name="dial_PWM">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>140</y>
         <width>50</width>
         <height>64</height>
        </rect>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="dSpinBox_Temp">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>210</y>
         <width>81</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>16</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>P</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>20</y>
         <width>16</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>I</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>20</y>
         <width>16</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>D</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>150</y>
         <width>41</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>PWM</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>150</y>
         <width>41</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Temp</string>
       </property>
      </widget>
     </widget>
     <zorder>groupBox</zorder>
     <zorder>listWidget</zorder>
     <zorder>pushButton_connect</zorder>
     <zorder>pushButton_link</zorder>
     <zorder>pushButton_send</zorder>
     <zorder>spinBox_DevNum</zorder>
     <zorder>textBrowser_usbMessage</zorder>
     <zorder>textBrowser_sendMessage</zorder>
     <zorder>pushButton</zorder>
     <zorder>pushButton_close1</zorder>
     <zorder>progressBar</zorder>
    </widget>
    <widget class="QWidget" name="tab_setup">
     <attribute name="title">
      <string>Select</string>
     </attribute>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>251</width>
        <height>251</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>249</width>
         <height>249</height>
        </rect>
       </property>
       <widget class="QRadioButton" name="radioButton">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>231</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Solidering </string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_bistate">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>241</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Bistate constant temp. controll</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_tristate">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>70</y>
          <width>241</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Tristate constant temp controll</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_tempCheck">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>100</y>
          <width>241</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Temperature check</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_tempCurve">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>130</y>
          <width>241</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>External temperature curve file</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>160</y>
          <width>241</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>PID temperature controll</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_select">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>50</y>
        <width>391</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>272</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Select type of work:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_close2">
      <property name="geometry">
       <rect>
        <x>568</x>
        <y>350</y>
        <width>91</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_loadFile">
      <property name="geometry">
       <rect>
        <x>568</x>
        <y>316</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Load file</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_plot">
     <attribute name="title">
      <string>Plot</string>
     </attribute>
     <widget class="QwtPlot" name="qwtPlot">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>29</y>
        <width>561</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_PlotData">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>30</y>
        <width>71</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_close3">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>340</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>676</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_close1</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>620</x>
     <y>422</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_connect</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>listview_populate_usb_devices()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_close2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>615</x>
     <y>432</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_close3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>632</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>listview_populate_usb_devices()</slot>
 </slots>
</ui>
